doc/configuration.rst

```diff
@@ -61,9 +61,15 @@ And some things can be tweaked directly in CSS:
 
 - ``.sphx-glr-thumbcontainer`` (:ref:`setting_thumbnail_size`)
 
-You can also use the package ``warnings`` in the ``conf.py`` file to prevent
-warnings from being captured and included in your built documentation. For
-example, to remove the specific Matplotlib agg warning, you can add::
+.. _removing_warnings:
+
+Removing warnings
+=================
+
+To prevent warnings from being captured and included in your built
+documentation, you can use the package ``warnings`` in the ``conf.py`` file.
+For example, to remove the specific ``UserWarning``: 'I don't want this
+warning', you can add::
```

Comment:
this text/explanation no longer matches what's done below

Reply:
Sorry, it is not my day today. Thanks!